charts/thoras/tests/__snapshot__/collector_deployment_test.yaml.snap

```diff
@@ -9,6 +9,8 @@ Default containers should match snapshots:
         value: "true"
       - name: bootstrap.memory_lock
         value: "false"
+      - name: path.data
```

Comment:
It's funny to me their env var convention doesn't include any indication what it's associated with. ES age showin hard through here (not that PG is younger 😂 but at least their env vars comply with best practices)